pkg/certsuite/certsuite.go

```diff
@@ -101,8 +101,15 @@ func processFlags() {
 	}
 }
 
+func isTnfSubcommand() bool {
+	cmd := strings.Split(os.Args[0], "/")
+	return cmd[len(cmd)-1] == "tnf"
+}
+
 func Startup() {
```

Comment:
Why not a parameter "initFlags bool" to startup to trigger initialization of flags? This way we don't need the isTnfSubcommand function

Reply:
Agree.